src/pat/checklist/checklist.js

```diff
@@ -79,15 +80,16 @@ define([
                 all_deselects = $(options.deselect);
             }
             for (var i=0; i<all_selects.length; i++) {
-
-                if (_._findSiblings(all_selects[i], "input[type=checkbox]:visible").filter(":not(:checked)").length === 0) {
+                siblings = _._findSiblings(all_selects[i], "input[type=checkbox]:visible");
+                if (siblings && siblings.filter(":not(:checked)").length === 0) {
```

Comment:
Not sure if this applies here as well but I often had to test explicitly for undefined, because any other value from undefined, even if 0 or empty string would make "siblings" true. Perhaps explicit is better than implicit here. (siblings != undefined &&…)

Reply:
Are you sure that 0 or '' will evaluate to `true`?
```
[ale@emily Patterns]$ node
> if (0) {console.log('I am true')};
undefined
> if (1) {console.log('I am true')};
I am true
undefined
```

Reply:
Ok, my case was different (https://github.com/Patternslib/Patterns/pull/547/files), here I got a jQuery object back. So this is good :)

Reply:
BTW that condition is really strange... a || !a && b should equal to a || b. It has a bug, it should be simplified or I am reading it wrong.